extensions/binding/org.eclipse.smarthome.binding.ntp/src/main/java/org/eclipse/smarthome/binding/ntp/handler/NtpHandler.java

```diff
@@ -96,8 +96,8 @@ public void initialize() {
 
 			Configuration config = getThing().getConfiguration();
 			this.hostname = (String) config.get(PROPERTY_NTP_SERVER);
-			this.refreshInterval = (BigDecimal) config.get(PROPERTY_REFRESH_INTERVAL);
-			this.refreshNtp = (BigDecimal) config.get(PROPERTY_REFRESH_NTP);
+			this.refreshInterval = new BigDecimal (config.get(PROPERTY_REFRESH_INTERVAL).toString());

```

Comment:
actually I don't quite understand how it got changed. From earlier I always understood the getThing().getConfiguration().get returned bigdecimals in case of numbers. Now it returned a Integer.
 
Note that the thing was created in discovery